src/frontend/http.rs

```diff
@@ -30,9 +79,33 @@ async fn get_file(
 ) -> Result<impl IntoResponse, AppError> {
     let pk = Secp256k1PubKey::try_from(pk.as_str())?;
     let blake3_hash = Blake3Hash(blake3::Hash::from_str(&blake3_hash)?);
+    let _file_bytes = read_file(&pk, &blake3_hash).await?;
```

Comment:
Why did you rename this? We lose type safety.

Reply:
That was for a temp scenario to validate the format of the file.
Removed.